sdk/scala-sdk/src/test/scala/com/akkaserverless/scalasdk/JsonSupportSpec.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * Copyright (C) 2009-2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.akkaserverless.scalasdk
+
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import com.akkaserverless.javasdk.{ JsonSupport => JavaJsonSupport }
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.annotation.JsonProperty
+
+@JsonCreator
+case class MyJsonable(@JsonProperty("field") field: String)
```

Comment:
Shouldn't need JsonProperty annotations when the DefaultScalaModule is added.
We should also add this to the build:
```
.settings(javacOptions += "-parameters") // for Jackson
```

Reply:
(and we should have that in samples build files)

Reply:
Added to sdk test compilation and scala samples